image/templates/helm/stackrox-secured-cluster/templates/_init.tpl.htpl

```diff
@@ -260,9 +260,9 @@ The following block checks for the validity of the provided init bundle. (`helm
 */}}
 {{ $operatorInitBundlePresent := hasKey (._rox.metadata).annotations "init-bundle.stackrox.io/name" }}
 {{ $helmInitBundleNotPresent := empty ._rox.ca._cert }}
-{{- if and ($operatorInitBundlePresent) ($helmInitBundleNotPresent) }}
-    {{ include "srox.fail" (list . "Error: It seems an init bundle in the operator format has been provided. Note that this bundle format is ignored by the Helm chart. Make sure to provide an init bundle for Helm chart installation.") }}
-{{- else if and (not ($helmInitBundleNotPresent)) ($operatorInitBundlePresent) }}
+{{- if and $operatorInitBundlePresent $helmInitBundleNotPresent }}
+    {{ include "srox.fail" (list . "FATAL: It seems an init bundle in the operator format has been provided. Note that this bundle format is ignored by the Helm chart. Make sure to provide an init bundle for Helm chart installation.") }}
+{{- else if and (not $helmInitBundleNotPresent) $operatorInitBundlePresent }}
     {{ include "srox.warn" (list . "It seems an init bundle in the operator format has been provided. Note that this bundle format is ignored by the Helm chart. Make sure to provide an init bundle for Helm chart installation.") }}
```

Comment:
I think this warning is a bit misleading. Because, the user *did* in fact provide an Helm init bundle in this case.
```suggestion
    {{ include "srox.warn" (list . "Any init bundle in the operator format you might have provided has been ignored.") }}
```

Reply:
@mclasmeier your suggested change is also misleading. In this case the user did in fact provide an operator-formatted init bundle (in you **might** have provided). I prefer the previous `It seems an init bundle in the operator format has been provided. Note that this bundle format is ignored by the Helm chart. Make sure to provide an init bundle for Helm chart installation.` as this indicates that 1. an operator init bundle was provided and 2. it tells the user that this bundle will be ignored.

Reply:
Not a native english speaker here.. I added the "might" simply because this is just heuristic to catch the situation.

That beign said, I don't have a strong opinion on the exact wording. I just think in *this* scenario we shouldn't tell the user `Make sure to provide an init bundle for Helm chart installation`, because that is most likely exactly what he or she did.

Reply:
absolutely (to both).